src/main/java/com/mealmentor/enterprise/dao/MealItemDAOStub.java

```diff
@@ -13,16 +13,15 @@
     HashMap<Integer, MealItem> allMealItems = new HashMap<>();
 
     @Override
-    public MealItem save(MealItem mealItem) throws Exception {
+    public MealItem save(MealItem mealItem) {
         Integer mealItemID = Integer.parseInt(mealItem.getMealItemId());
         allMealItems.put(mealItemID, mealItem);
         return mealItem;
     }
 
     @Override
     public List<MealItem> fetchAll() {
-        List<MealItem> returnsMealItems = new ArrayList<>(allMealItems.values());
-        return returnsMealItems;
+        return new ArrayList<>(allMealItems.values());
```

Comment:
No need to declare a variable, just return the product of the method call. This also happened a few times in the code.

Reply:
Good.  Save some lines.